build.gradle

```diff
@@ -72,6 +72,7 @@ ext {
 
     argparse4jVersion = '0.7.0'
     junitVersion = '4.12'
+    evaluatorVersion = '4.0.0-SNAPSHOT'
```

Comment:
Won't this be a problem? `Neo4j 4.0.0-SNAPSHOT` is not on maven central, so how is anyone going to build cypher-shell now, without building some local snapshot of Neo4j?  

It would be great if we could get a release of the evaluator somehow.

Reply:
yes we will have to figure that out. This PR is mostly to check the integration with the evaluator.